### ISSUE TITLE
Add more values (columns) to tooltips

### ISSUE THREAD

**Issue by ivan-aksamentov** [hodcroftlab/covariants#29]
We might want to add more helpful information to tooltips.

For example, we currently only show frequencies in country plots. But we can also show current and totals, by adding 1 more columns.

Should be approved by a scientist

**Comment by emmahodcroft** [comment]
I think it's the variant plots where show frequencies only, and countries we show counts only - but agree that we could add another column to each showing the other!

**Comment by ivan-aksamentov** [comment]
@emmahodcroft in my example I meant something like

Country | Frequency | Current | Total |
--------|-----------|---------|-------|
France |     0.5    |    10     |  20  | 
Italy  |     0.33   |    3     |   10 |

to show all the values we have, not only the computed one (frequency)

But, yes the idea is that we can basically show anything in tooltips. It does not even have to be a table, or not only a table. It's just another piece of HTML.

**Comment by emmahodcroft** [comment]
Yes - I probably didn't express myself very well, but I agree, this would be cool! I just meant that in one of the pages we currently show counts and the other we currently showing frequencies, but we could show both in both.

**Comment by emmahodcroft** [comment]
Updating this - I'm pretty happy with what we have for Per Country, but we could add a third column showing the total number of sequences for that country (or up to that point). Not sure it's helpful though.

For per Variant if I can switch to using actual counts in 2-week intervals instead of smoothed 1-week, we could add 'real numbers' to that as well.

**Comment by AdvancedCodingMonkey** [comment]
We settled on making the per-variant tooltip similar to the other tooltips by adding the number of sequences and a total at the bottom.
